plugins/connection/network_cli.py

```diff
@@ -269,6 +269,21 @@
           key: ssh_type
     vars:
     - name: ansible_network_cli_ssh_type
+  host_key_checking:
+    description: 'Set this to "False" if you want to avoid host key checking by the underlying tools Ansible uses to connect to the host'
+    type: boolean
+    default: True
+    env:
+    - name: ANSIBLE_HOST_KEY_CHECKING
+    - name: ANSIBLE_SSH_HOST_KEY_CHECKING
+    ini:
+    - section: defaults
+      key: host_key_checking
+    - section: persistent_connection
+      key: host_key_checking
+    vars:
+    - name: ansible_host_key_checking
+    - name: ansible_ssh_host_key_checking
```

Comment:
don't you need the paramiko ones also?

Reply:
People using network_cli connection most likely won't be referring to paramiko plugin options so IMO it is not required. Alternately if we decide to add paramiko options we will also require to add options of libssh plugin.

Reply:
iiuc, if we add both _ibssh and paramiko options here, the behavior might be a bit misleading, i.e., setting `ansible_paramiko_host_key_checking: False`, would turn of host key checking even when libssh is used underneath, and vice versa.